docs/cloud/alerts-notifications/add-discord-notification.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Add discord notification configuration"
+sidebar_label: "Add discord notification configuration"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/add-discord-notification-configuration.md"
+sidebar_position: "1"
+learn_status: "Published"
+learn_topic_type: "Tasks"
+learn_rel_path: "Operations/Alerts"
+learn_docs_purpose: "Instructions on how to add notification configuration for discord"
+-->
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add specific configuration to get notifications delivered on discord.
+
+#### Prerequisites
+
+To add discord notification configurations you need
+
+- A Cloud account
+- Access to the space as and Administrator
+- Have your discord server able to receive webhook integrations, for mode details check [how to configure this on discord](#settings-on-discord)
+
+#### Steps
+
+1. Click on the **Space settings** cog (located above your profile icon)
+1. Click on the **Notification** tab
```

Comment:
```suggestion
2. Click on the **Notification** tab
```

Reply:
Enumerating lists like this is acceptable both on GH and in learn. We dont have a concrete guideline about this but it works and removes confusion because the person who writes the doc dont have to think about keeping the order of the enumeration.